assets/js/components/dashboard-sharing/UserRoleSelect.test.js

```diff
@@ -102,7 +102,7 @@ describe( 'CurrentSurvey', () => {
 			}
 		);
 
-		await waitForRegistry();
+		await act( () => waitForRegistry() );
```

Comment:
This shouldn't be needed because `waitForRegistry` uses `act` internally.

https://github.com/google/site-kit-wp/blob/189863daf2e61418eedb7f73ac22badbfc2ef20a/tests/js/test-utils.js#L117

It's used many other times in this file as well without it so this doesn't make sense as a one-off. If there is a specific reason why it's needed here, please say why. The problem is probably just that it can't guarantee that the registry is done updating before proceeding and the now async resolvers (particularly those that invoke each other) may make this less stable than it was before.

Reply:
Yeah, oddly it kept failing in CI. Okay, I'll take a look and see what's up, but I figured something in the fixture data was being odd 🤷🏻